docs/getting-started.md

```diff
@@ -9,65 +9,234 @@ sidebar_label: Getting started
+EACCES: permission denied
+```
+
+It's probably because you originally installed Node with root permissions [explain what this means] . Because of this, writing to your npm directory (```npm -i -g```) requires root permissions too.
+
+While it's not a good idea to have Node installed this way, one way to quickly give yourself root permissions is to run the slightly modified command: [maybe not a good idea to show this option]
+
+```sh
+sudo npm i -g --unsafe-perm @aragon/cli
+```
+
+An arguably better way to fix the problem is to follow the steps outlined in this [stackoverflow answer.](https://stackoverflow.com/a/24404451) [should make this more intuitive...]
 
-You might need to have [Git](https://git-scm.com) installed.
+[i think this whole section needs to be exlained more simply...]
 
 ### Windows considerations
 
-You might need to run the shell with administrator rights when installing the aragonCLI, because our `go-ipfs` dependency will need to create a symlink to work correctly.
+You might need to run the shell with administrator rights when installing the aragonCLI, because our `go-ipfs` dependency will need to create a symlink to work correctly. [this needs to be explained better]
 
-If you have problems during the instalation of aragonCLI or any other dependencies. You probably need to install [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) or similar package.
+If you have problems during the instalation of aragonCLI or any other dependencies. You probably need to install [windows-build-tools](https://www.npmjs.com/package/windows-build-tools) or similar package. 
 
+Again, if you're having trouble fixing things, please reach out to us at the [#dev-help channel on the Aragon Chat](https://aragon.chat/channel/dev-help)
+
+### Note on Git
+
+You might need to have [Git](https://git-scm.com) installed. [a little startling for the reader... why might? i think we need to elaborate here...]
 
 ## Quick start
 
-To create your first DAO run:
+In order to get up and running quickly, we’ll build our first DAO using some basic scaffolding [explain what scaffolding means] -- remember a DAO is just a combination of Aragon Apps... [elaborate...]
+
+To create our DAO run:
 
 ```sh
 npx create-aragon-app foo.aragonpm.eth
 cd foo
 npx aragon run
 ```
 
-Congrats you have just created a DAO! It’s running on your local network and as soon as it's ready it will open in your browser at [localhost:3000](http://localhost:3000)!
+Don't worry about understanding the first line right now. Really. We'll cover that in the [tutorial](https://hack.aragon.org/docs/tutorial.html) coming up.
+
+With respect to the third line, ```aragon run``` just sets up everything for us in the background so we can quickly live test our DAO.
+
+N.B. If you're unsure what the difference is between ```npx``` and ```npm```, we recommend you read through this [stackoverflow post.](https://stackoverflow.com/questions/50605219/difference-between-npx-and-npm)
```

Comment:
Other good resource we could link here is [this post](https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b)

Reply:
Done